common/source/docs/common-canbus-setup-advanced.rst

```diff
@@ -50,6 +50,8 @@ parameter ``CAN_Px_DRIVER``, where x is the number of the CAN port.
 The value of this parameter is the id of driver that will be associated with this
 port (interface).
 
+Each enabled bus/driver will use 12k of memory (RAM not flash) with defaults.
```

Comment:
actually the total pool is 16K by default....and serves all drivers...so two enabled CAN ports will be allocated 8K